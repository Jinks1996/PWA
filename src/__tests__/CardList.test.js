import { shallow, mount, render } from "enzyme";
import CardList from "../components/CardList";

// snapshot testing
it("expects to render CardList Component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "John",
      email: "john@gmail.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
