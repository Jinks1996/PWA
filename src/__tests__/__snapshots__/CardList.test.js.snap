// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render CardList Component 1`] = `ShallowWrapper {}`;
